settings.gradle.kts

```
@@ -7,5 +7,6 @@ pluginManagement {
         id("com.github.sgtsilvio.gradle.utf8") version "${extra["plugin.utf8.version"]}"
         id("com.github.sgtsilvio.gradle.metadata") version "${extra["plugin.metadata.version"]}"
         id("com.github.sgtsilvio.gradle.javadoc-links") version "${extra["plugin.javadoc-links.version"]}"
+        id("io.github.gradle-nexus.publish-plugin") version "${extra["plugin.io.github.gradle-nexus.publish-plugin.version"]}"
```

```suggestion
        id("io.github.gradle-nexus.publish-plugin") version "${extra["plugin.nexus-publish.version"]}"
```
(I like to keep it readable)